components/playground.tsx

```diff
@@ -80,16 +151,17 @@ export default function Playground() {
               apis.some((api) => !api.api_host) &&
               !mockApiResponses)
               ? `You need to add${numActions ? "" : " actions (Actions tab)"}${
-                  numActions ||
-                  (apis?.length && apis.some((api) => !api.api_host)) ||
-                  mockApiResponses
+                  !numActions &&
+                  ((apis?.length && apis.some((api) => !api.api_host)) ||
+                    !mockApiResponses)
```

Comment:
something up with the string here:

![image](https://github.com/Superflows-AI/superflows/assets/28537547/c341ade9-3dad-4843-88cd-943799e5fd8e)

Reply:
I noticed we aren't checking for an api key, is it possible to have an API that doesn't require a key?